practices/CervantesAlexis/practica03/practica3.py

```diff
@@ -0,0 +1,205 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+import numpy as np
+import pandas as pd
+import os
+import re
+from PIL import Image
+from os import path
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+from nltk.corpus import stopwords
+import nltk
+import random
+import string
+from collections import Counter
+
+corpus_freqs = pd.read_csv('crea_full.csv', delimiter='\t', encoding="latin-1")
+
+freq_dict = corpus_freqs.set_index('words')['freq'].to_dict()
+
+nltk.download('stopwords')
+
+stopwords = stopwords.words('spanish')
+
+freq_stop = {}
+for i in stopwords:
+    if i in freq_dict.keys():
+        freq_stop[i] = freq_dict[i]
+
+
+def makeImage(text):
```

Comment:
El estandar de python es snake_case no CamelCase para funciones. Sugiero seguir el estilo de python
```suggestion
def make_image(text):
```